recipes-bsp/u-boot/u-boot_2019.07.bbappend

```diff
@@ -39,6 +41,11 @@ do_configure_prepend_freedom-u540() {
     fi
 }
 
+do_compile_append_freedom-u540_sota() {
+    sed -e 's/@@KERNEL_BOOTCMD@@/${KERNEL_BOOTCMD}/' \
+        "${WORKDIR}/uEnv.txt.in" > uEnv.txt
```

Comment:
This was only done for the recipe to be compatible with multiple targets, and not really required here.

If you just rename uEnv.txt.in to uEnv.txt and install that file it should already be enough for what you need.

Reply:
I like the idea of substitution and have a single file to generate output